agent/containers/images/apply-tags

```diff
@@ -13,7 +13,7 @@ fi
 githash="$(grep -v -E "^v" ${tags_file})"
 
 function apply_tag {
-    buildah tag ${1}:${githash} ${1}:${2}
+    ${BUILDAH_ECHO} buildah tag ${1}:${githash} ${1}:${2}
```

Comment:
Where does `BUILDAH_ECHO` come from (I don't see it in `main`)? I assume the intent is something along the lines of `BUILDAH_ECHO=echo make` to disable the container commands for testing, but a descriptive comment somewhere would seem appropriate.

Reply:
As you surmise, it is intended to allow for fast-turnaround, low-impact testing.

By default, `BUILDAH_ECHO` is undefined, and so it expands vacuously with no effect, leaving the `buildah` commands to execute normally; however, if it is defined to `echo`, then, instead of executing the commands, they are simply echoed.

I think there is a better way to accomplish this, but it would be more effective if the `buildah` commands were placed in the `make` recipes instead of wrapped inside external scripts; so, for the moment, this approach is worth adopting.

Reply:
I'd prefer to have this at least recorded in a comment somewhere, because it looks rather weird. But "whatever" ... I'll resolve this.